### PR TITLE
issue_1140: enable creating css locators

### PR DESCRIPTION
**Pull request by arslanbekova**

### CODE REVIEW

File: src/features/locators/components/LocatorEditDialog.tsx [hunk 1]
```diff
@@ -119,9 +115,10 @@ export const LocatorEditDialog: React.FC<Props> = ({
     const locatorMessage = isLocatorFieldTouched ? validationMessage : LocatorValidationWarnings.NotFound;
 
     const { name, type, locator, locatorType } = await form.validateFields();
+    const isCSSLocator = locatorType === LocatorType.cssSelector;
     newLocator = {
       ...newLocator,
-      locator: { ...newLocator.locator, xPath: locator }, // revise when enable creating css locator
+      locator: { ...newLocator.locator, ...(isCSSLocator ? { cssSelector: locator } : { xPath: locator }) },
```
Comment (MariiaNebesnova) [inline src/features/locators/components/LocatorEditDialog.tsx:121]: let me suggest a shorter solution: 
```
locator: { ...newLocator.locator}, ...{ [isCSSLocator ? "cssSelector" : "xPath"]: locator }
```

File: src/features/locators/reducers/addCustomLocator.thunk.ts [hunk 2]
```diff
@@ -24,14 +32,20 @@ export const addCustomLocator = createAsyncThunk(
...
+          ({ foundHash, foundElementText } = JSON.parse(await evaluateCssSelector(locator.cssSelector, element_id)));
+        }
 
         if (!foundHash) {
           foundHash = element_id.split("_")[0];
           await sendMessage
-            .assignJdnHash({ jdnHash: foundHash, locator: locator.xPath, isCSSLocator: false })
+            .assignJdnHash({
+              jdnHash: foundHash,
+              ...(isCSSLocator ? { locator: locator.cssSelector } : { locator: locator.xPath }),
```
Comment (MariiaNebesnova) [inline src/features/locators/reducers/addCustomLocator.thunk.ts:46]: same suggestion

File: src/features/locators/reducers/addCustomLocator.thunk.ts [hunk 3]
```diff
@@ -24,14 +32,20 @@ export const addCustomLocator = createAsyncThunk(
 
     if (getLocatorValidationStatus(message) === ValidationStatus.SUCCESS) {
       try {
-        const result = JSON.parse(await evaluateXpath(locator.xPath, element_id));
-        const { foundElementText } = result;
-        let { foundHash } = result;
+        if (!isCSSLocator) {
```
[the remark this answers is not in the document]
Comment (arslanbekova) [reply src/features/locators/reducers/addCustomLocator.thunk.ts]: fixed

**Review by MariiaNebesnova — APPROVED** [review]
Well done! Thank you!

**Pull request merged**